packages/gestalt/src/Image.js

```diff
@@ -152,50 +153,42 @@ export default class Image extends PureComponent<Props> {
       srcSet,
     } = this.props;
 
-    const isScaledImage = shouldScaleImage(fit);
     const childContent = children ? (
       <Box position="absolute" top left bottom right overflow="hidden">
         {children}
       </Box>
     ) : null;
 
-    return isScaledImage ? (
-      <Box height="100%" position="relative">
-        <div
-          aria-label={role === 'presentation' ? undefined : alt}
-          className={fit === 'contain' || fit === 'cover' ? styles[fit] : null}
-          style={{
-            backgroundColor: color,
-            backgroundImage: `url('${src}')`,
-          }}
-          role={role}
-        />
-        {childContent}
-      </Box>
-    ) : (
+    const isScaledImage = fit === 'cover' || fit === 'contain';
+    const fitStyles = fit === 'cover' || fit === 'contain' ? styles[fit] : undefined;
```

Comment:
`isScaledImage ` to be used here?

Reply:
<img width="291" alt="Screen Shot 2022-11-08 at 2 14 25 PM" src="https://user-images.githubusercontent.com/10593890/200654632-d80e8467-e0be-4deb-9e12-1d6cbeab3288.png">


Does `background-size` work in <img>. it was being applied to div, not sure if it works with <img>

Is it redundant with `objectFit` /  dead code?

Reply:
> `isScaledImage ` to be used here?

Flow doesn't realize that "none" won't be an option if we use that variable, unfortunately

Reply:
> Does background-size work in . it was being applied to div, not sure if it works with 
> Is it redundant with objectFit / dead code?

Good call — was able to simplify the styles a bit